tests-integ/test_model_openai.py

```diff
@@ -0,0 +1,46 @@
+import os
+
+import pytest
+
+import strands
+from strands import Agent
+from strands.models.openai import OpenAIModel
+
+
+@pytest.fixture
+def model():
+    return OpenAIModel(
+        model_id="gpt-4o",
+        client_args={
+            "api_key": os.getenv("OPENAI_API_KEY"),
+        },
+    )
+
+
+@pytest.fixture
+def tools():
+    @strands.tool
+    def tool_time() -> str:
+        return "12:00"
+
+    @strands.tool
+    def tool_weather() -> str:
+        return "sunny"
+
+    return [tool_time, tool_weather]
+
+
+@pytest.fixture
+def agent(model, tools):
+    return Agent(model=model, tools=tools)
+
+
+@pytest.mark.skipif(
+    "OPENAI_API_KEY" not in os.environ,
```

Comment:
What's the status of getting keys for these?  Do we all have keys that we can share for test purposes?

Reply:
If we do have keys, can we put them into our test AWS account secrets-manager (doesn't block this PR)